pkg/update/versioner.go

```diff
@@ -135,7 +139,7 @@ func (g GitHub) getReleaseID(ctx context.Context, version semver.Version) (id in
 }
 
 func (g GitHub) getAssetID(assets []github.ReleaseAsset, version semver.Version) (id int64, err error) {
-	target := fmt.Sprintf("fastly_v%s_%s-%s.tar.gz", version, runtime.GOOS, runtime.GOARCH)
+	target := fmt.Sprintf("%s_v%s_%s-%s.tar.gz", g.org, version, runtime.GOOS, runtime.GOARCH)
```

Comment:
`fastly` here is possibly not a github org

Reply:
Agreed, `fastly` in this context is the name of the binary program not the org. This is also applicable to many references above in the `LatestVersion()` method above, thus similar comments have been elided.

Furthermore, the filename pattern `fastly_v0.26.2_linux-amd64.tar.gz` is tightly coupled to the CLI release implementation and thus very opinionated and might not be the pattern that Viceroy or other future binaries use. Easy option is we ensure Viceroy uses this pattern or we will have to give some thought and refactoring as to how we generate the asset filename.

Reply:
@cratelyn 👋🏻 would you be able to offer any insights here ^^ re: filename pattern. Thanks.

Reply:
I think that this filename pattern sounds okay to me. Using semantic versioning would be fine, and putting a target/arch at the end is definitely worthwhile. No objections here!